proxy/src/main/java/com/velocitypowered/proxy/protocol/packet/chat/session/SessionCommandHandler.java

```diff
@@ -52,6 +54,10 @@ public void handlePlayerCommandInternal(SessionPlayerCommand packet) {
               "A proxy plugin caused an illegal protocol state. "
                   + "Contact your network administrator."));
         }
+        // We seemingly can't actually do this if signed args exist, if not, we can probs keep stuff happy
+        if (player.getProtocolVersion().getProtocol() == ProtocolVersion.MINECRAFT_1_20_2.getProtocol()) {
```

Comment:
Be optimistic that the packet is there in 1.20.3 too.
```suggestion
        if (player.getProtocolVersion().compareTo(ProtocolVersion.MINECRAFT_1_20_2) >= 0) {
```